config.yaml

```diff
@@ -259,7 +259,91 @@ languages:
       - name: ブログ
         url: "/blog"
         weight: 5
-       
+
+  kr:
```

Comment:
Also, as soon PR #178 gets merged in, the Korean translation of config.yaml file have to be put in its own file under `data/ko/config.yaml`

Reply:
got it ill get the changes in soon